frontend/src/scenes/session-recordings/components/InternalSurvey/InternalMultipleChoiceSurvey.tsx

```diff
@@ -0,0 +1,277 @@
+/**
+ * @fileoverview A component that displays an interactive survey within a session recording. It handles survey display, user responses, and submission
+ */
+import { LemonButton, LemonCheckbox, LemonTextArea, Link } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { maxThreadLogic } from 'scenes/max/maxThreadLogic'
+import { FEATURE_FLAGS } from 'lib/constants'
+
+import { SidePanelTab, SurveyQuestion, SurveyQuestionType } from '~/types'
+
+import { internalMultipleChoiceSurveyLogic } from './internalMultipleChoiceSurveyLogic'
+import MaxTool from 'scenes/max/MaxTool'
+import { maxLogic } from 'scenes/max/maxLogic'
+import { sidePanelSettingsLogic } from '~/layout/navigation-3000/sidepanel/panels/sidePanelSettingsLogic'
+
+import { featureFlagLogic } from 'lib/logic/featureFlagLogic'
+
+interface InternalSurveyProps {
+    surveyId: string
+}
+
+function TooExpensiveHelpMessage(): JSX.Element {
+    return (
+        <>
+            <p>
+                We aim to be significantly cheaper than our competitors. In addition to our pay-as-you-go pricing, below
+                are few ways to reduce costs by recording fewer sessions:
+                <ul className="list-disc pl-4 text-secondary">
+                    <li>Disable automatic recording and programmatically start and stop recordings</li>
+                    <li>
+                        Set a feature flag with the conditions that define which users or sessions you want to record.
+                    </li>
+                    <li>Set a minimum recording duration.</li>
+                    <li>Set a sampling rate for recordings.</li>
+                    <li>
+                        Make sure you capture at least one event per session to reduce the costs while maintaining all
+                        filtering options.
+                    </li>
+                </ul>
+            </p>
+            <Link to="https://posthog.com/docs/session-replay/cutting-costs" target="_blank">
+                Learn more
+            </Link>
+        </>
+    )
+}
+
+function PrivacyHelpMessage(): JSX.Element {
+    return (
+        <>
+            <p>
+                PostHog offers a range of controls to limit what data is captured by session recordings. Our privacy
+                controls run in the browser or mobile app. So, masked data is never sent over the network to PostHog.
+            </p>
+            <Link to="https://posthog.com/docs/session-replay/privacy" target="_blank">
+                Learn more
+            </Link>
+        </>
+    )
+}
+
+function PerformanceHelpMessage(): JSX.Element {
+    return (
+        <>
```

Comment:
without commenting on them all
we should put data-atts on all of these containers so we can check when/if people are interacting with them